SECURITY.md

```diff
@@ -1,5 +1,174 @@
+
+Impacts only apply to assets in active use by Sui.
+
+### Blockchain/DLT
+
+-   Sui Network Consensus
+-   Sui Network
+-   Sui Move
+
+### Smart Contract
+
+-   Sui Framework
+
+## Impacts in scope
+
+Only the following impacts are accepted within this bug bounty program. All other impacts are considered out-of-scope and ineligible for payout.
+
+### Coin and object integrity bugs
+
+CRITICAL [$500,000 USD]
+
+-   Exceeding the maximum token supply of 10B SUI and allowing attacker to claim the excess funds
+-   Unauthorized creation, copying, transfer or destruction of objects via bypass of or exploit of bugs in the Move or Sui bytecode verifier
+-   Address collision—creating two distinct authentication schemes that hash to the same SUI address
+-   Object ID collision—creating two distinct objects with the same ID, with financial consequences
+
+MEDIUM [$10,000 USD]
+
+-   Going under the maximum supply of SUI by sending a transaction that burns SUI that will not be re-minted into validator rewards or the storage fund at at the next epoch
+
+LOW [$5,000 USD]
+
+-   Sending a transaction that triggers invariant violation error code in unmodified validator software
+
+Other vulnerabilities leading to theft or loss of valuable objects will have their severity determined depending upon the consequences of the issue and the preconditions for exploiting it.
+
+### Bypassing authentication
+
+CRITICAL [$500,000 USD]
+
+-   Including an owned object as a transaction input without a valid signature from the object’s owner, in a manner that leads to significant loss of funds
+-   Dynamically loading an object that is not directly or transitively owned by the transaction sender, in a manner that leads to significant loss of funds
+-   Unauthorized upgrade of a Move package, in a manner that leads to significant loss of funds
+-   Locking an owned object without a valid signature from the object’s owner
+
+### Staking
+
+CRITICAL [$500,000 USD]
+
+-   Acquiring voting power vastly disproportionate to stake, not including:
+    -   Voting power that is redistributed because one or more other validators already has max voting power
+    -   Rounding errors that result in minor voting power discrepancies
+-   Approve a transaction or checkpoint with less than ⅔ voting power
+-   Stealing staking rewards that belong to another user, or claiming more than a user’s share of staking rewards, not including
+    -   Rounding errors that result in a minor, financially insignificant discrepancy
+
+Any other issue leading that undermines proof of stake governance or staking rewards, with severity depending on the consequences of the issue and the preconditions for exploiting it
+
+### Operational
+
+CRITICAL [$500,000 USD]
+
+-   Cause an unrecoverable network halt that must be fixed via a hard fork
+-   Arbitrary, non-Move remote code execution on unmodified validator software
+-   Send a transaction that causes an uncaught panic in unmodified validator software
```

Comment:
what does this mean exactly? does this refer to any panic?

Reply:
We've updated the copy, please re-review @dariorussi